Highcard.rb

```diff
@@ -0,0 +1,69 @@
+require
+
+# initialize empty deck
+# build a deck
+# loop though ranks
+# .each with index -> to pull the worth of the card
+# loops suits
+# push each pair into the deck
+# shuffle the deck
+
+
+ranks = [2, 3 , 4, 5 ,6 ,7 ,8 ,9 ,10 , "A" "J" , "Q" , "K"]
+
+suits = ["hearts" , "spades" , "clubs" , "diamonds"]
+
+
+$players = []
```

Comment:
how could you wrap this functionality in a class, and make this an attribute of that object to avoid utilizing global vars in Ruby?